pymbar/pmf.py

```diff
@@ -454,13 +454,19 @@ def generate_pmf(
 
         return result_vals  # should we return results under some other conditions?
 
-    def _setup_pmf_histogram(histogram_parameters):
+    def _setup_pmf_hiostogram(histogram_parameters):
```

Comment:
Methods require a first argument `self` _or_ decorating them with `@staticmethod` if they don't need access to `self`.

```suggestion
    def _setup_pmf_histogram(self, histogram_parameters):
```
or 

```suggestion
    @staticmethod
    def _setup_pmf_histogram(histogram_parameters):
```

In this case you do need `self`, so we can only choose option A.

Reply:
I think I addressed this in most recent commit - the first one was just spewing text up.  I'll go through and look for ones that don't access self and add the decorator.

Reply:
And spelling error fixed.